ee/clickhouse/client.py

```diff
@@ -74,6 +85,18 @@ def async_execute(query, args=None):
         connections_max=100,
     )
 
+    def cache_sync_execute(query, args=None, redis_client=None, ttl=CACHE_TTL):
+        if not redis_client:
+            redis_client = redis.get_client()
+        key = _key_hash(query, args)
```

Comment:
Nitpick: We're serializing the data twice here - given the data can be quite large let's do it once.

Reply:
I'm confused, where are is it serializing twice?

Reply:
_serialize call and _key_hash both do the same thing

Reply:
I am stupid. Ignore me.